insights/config/specs.py

```diff
@@ -373,6 +373,7 @@
     "teamdctl_state_dump"       : CommandSpec("/usr/bin/teamdctl {iface} state dump", iface=r"team\S+"),
     "tomcat_web.xml"            : First([PatternSpec(r"etc/tomcat.*/web\.xml"),
                                     PatternSpec(r"conf/tomcat/tomcat.*/web\.xml")]),
+    "tomcat_virtual_dir_context": CommandSpec("/bin/grep -R --include '*.xml' 'VirtualDirContext' /usr/share/tomcat*"),
```

Comment:
For the `master` branch, we'd better add the specs to both the `insights/config/specs.py` and `insights/specs.py`.

Sorry for missing this in the first glance.